src/main/java/frc/robot/commands/ShooterTuner.java

```diff
@@ -34,14 +34,14 @@ public ShooterTuner(Shooter shooter, Carousel carousel, CarouselCommand cc) {
   @Override
   public void initialize() {
     shooter.calibratePID(0.000145, 1e-8, 0, 6.6774 * 0.00001);
-    shooter.setLEDRing(true);
+    shooter.vision.setLEDRing(true);
     SmartDashboard.putNumber("Distance", SmartDashboard.getNumberArray("vision/target_info", new double[]{0,0,0,0,0,0,0})[3]);
```

Comment:
Then this would be SmartDashboard.putNumber("Distance", shootervision.getDistance())

Reply:
I tried but that didn't work with the putNumber method since its second parameter needs to be an array of doubles and getDistance just returns a double